spec/api/configuration/retrieve_spec.rb

```diff
@@ -16,13 +17,21 @@
     end
 
     before(:each) do
+      ::Afterpay.configure do |c|
+        c.user_agent = user_agent
+      end
       stub_request(:get, %r{api.us.afterpay.com/v2/configuration})
+        .with(headers: { "User-Agent" => user_agent })
```

Comment:
I think you can omit the `with` part here cause you are testing it in the `makes request with the correct headers with the user_agent` spec

Reply:
True!